pkg/builtins/output.go

```diff
@@ -0,0 +1,41 @@
+package builtins
+
+import "github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+
+const OUTPUT_BUILTIN_NAME = "output"
+
+type OutputBuiltinRunner struct {
+	base     memory.Relocatable
+	included bool
+}
+
+func NewOutputBuiltinRunner(included bool) *OutputBuiltinRunner {
+	return &OutputBuiltinRunner{
+		base:     memory.NewRelocatable(0, 0),
+		included: included}
+}
```

Comment:
```suggestion
func NewOutputBuiltinRunner(included bool) *OutputBuiltinRunner {
	return &OutputBuiltinRunner{
		included: included}
}
```
zero-values can be omitted during initialization